avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1591,20 +1594,28 @@ function Set-UsageExamplesSection {
                 }
             }
 
-            # [5/6] Convert Bicep parameter block to JSON parameter block to enable processing
+            # [4/6] Convert Bicep parameter block to JSON parameter block to enable processing
             $conversionInputObject = @{
                 BicepParamBlock = ($paramsBlockArray | Out-String).TrimEnd()
                 CurrentFilePath = $testFilePath
             }
             $paramsInJSONFormat = ConvertTo-FormattedJSONParameterObject @conversionInputObject
 
-            # [6/6] Convert JSON parameters back to Bicep and order & format them
+            # [5/6] Convert JSON parameters back to Bicep and order & format them
             $conversionInputObject = @{
                 JSONParameters         = $paramsInJSONFormat
                 RequiredParametersList = $RequiredParametersList
             }
             $bicepExample = ConvertTo-FormattedBicep @conversionInputObject
 
+            # [6/6] Convert the Bicep format to a Bicep parameters file format
+            $bicepParametersFileExample = $bicepExample -split '\r?\n' | ForEach-Object {
+                $line = $_
+                $line = $line -replace '^(\s{4})([a-zA-Z]*)(:)(.*)', 'param $2 =$4' # Update any [    xyz: abc] to [param xyz = abc]
```

Comment:
I'm not sure if we want to hardcode an indent of exactly 4 here. If you check for any reference of the word `indent` in this file, you'll find I tried to instead identify the top-level indent dynamically like this:
```powershell
$bicepParamBlockArray = $BicepParamBlock -split '\n'
$topLevelParamIndent = ([regex]::Match($bicepParamBlockArray[0], '^(\s+).*')).Captures.Groups[1].Value.Length
```
By the time you hit this line I wouldn't know how the object `bicepExample` looks like without debugging, but if possible, please consider building the dynamic element in

Reply:
Good suggestion. Done :-)

Reply:
Nothing else to add from my side. Thanks